cmd/create/machinepool/cmd.go

```diff
@@ -233,6 +248,52 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	// Single AZ machine pool for a multi-AZ cluster
+	var multiAZMachinePool bool
+	var availabilityZone string
+	if cluster.MultiAZ() {
+		if !isMultiAvailabilityZoneSet {
+			multiAZMachinePool, err = interactive.GetBool(interactive.Input{
```

Comment:
unless isAvailabilityZoneSet, in which case you an assume multiAZMachinePool=false

Reply:
Changed, the user can choose a single AZ machine pool implicitly - setting only the `availabiltiy_zone` flag.
![image](https://user-images.githubusercontent.com/57869309/172600377-aea65a20-140f-4e7b-b3e6-5c38be3fc097.png)